Add override troubleshooting in README (#373)
stolostron/governance-policy-addon-controller: README.md

persist, but direct changes to resources on a managed cluster will still be reverted to match the
ManifestWork.

> ### Image Override Troubleshooting
> If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

Reviewer: Could you also add newlines before and after the paragraph and add some line wrapping? It matches formatting elsewhere here. (And I think VSCode might also do this for you if you use the Prettier extension.)
```suggestion
### Image Override Troubleshooting

If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

```

Author: Prettier for readme?

Reviewer: Yes--Prettier for Markdown, specifically.

persist, but direct changes to resources on a managed cluster will still be reverted to match the
ManifestWork.

> ### Image Override Troubleshooting
> If there is a trouble to override an image in ACM(ex: Daily cluster), `klusterletaddonconfig-pause` annotation is required to override `addon.open-cluster-management.io/values` in `ManagedClusterAddon`. 

Reviewer: ```suggestion
### Image Override Troubleshooting

If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

```

Author: Thanks!

```shell
oc annotate klusterletaddonconfig -n ${CLUSTER_NAME} ${CLUSTER_NAME} klusterletaddonconfig-pause=true --overwrite=true
```

> 

Reviewer: ```suggestion
```

persist, but direct changes to resources on a managed cluster will still be reverted to match the
ManifestWork.

### Image Override Troubleshooting

> If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

Reviewer: Sorry, one final thing is line wrapping so this isn't a super long line:
```suggestion
If there is trouble overriding an image or other configurations in ACM, the
`klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
`addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.
```

persist, but direct changes to resources on a managed cluster will still be reverted to match the
ManifestWork.

### Image Override Troubleshooting

> If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the

Reviewer: One more line break and it's good to go! 😬 

```suggestion
If there is trouble overriding an image or other configurations in ACM, the 
`klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
```

Author: Is it correct?
